DualityEditor/Utility/Extensions/ExtMethodsDualityFont.cs

```diff
@@ -113,12 +114,25 @@ public static void RenderGlyphs(this DualityFont font, FontFamily fontFamily)
 		/// This method assumes that the system font's size and style match the one specified in
 		/// the specified Duality font.
 		/// </summary>
-		private static void RenderGlyphs(DualityFont target, SysDrawFont internalFont)
+		private static void RenderGlyphs(DualityFont target, SysDrawFont internalFont, ExtendedCharSet extendedSet = null)
 		{
-			DualityFont.GlyphData[] glyphs = new DualityFont.GlyphData[DefaultChars.Length];
+            string allChars = DefaultChars;
+            string allCharBaseLineRef = CharBaseLineRef;
+            string allCharDescentRef = CharDescentRef;
+            string allCharBodyAscentRef = CharBodyAscentRef;
+
+            if(extendedSet != null)
+            {
+                allChars += extendedSet.Chars;                      //Maybe we should check if any char is duplicated
+                allCharBaseLineRef += extendedSet.CharBaseLine;     //And also check availbility of those characters in current font
```

Comment:
Code Style: Avoid comments at the end of a code line. Also add a space after `//` before the text begins.